glam/api/migrations/0022_auto_20220303_1530.py

```diff
@@ -0,0 +1,63 @@
+# Generated by Django 3.1.13 on 2022-03-03 15:30
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('api', '0021_fog_counts'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='FOGAggregationView',
```

Comment:
@alekhyamoz why do we need this model?

Reply:
Not required. was included as part of testing